packages/tenant-process/src/services/tenantService.ts

```diff
@@ -324,7 +325,7 @@ export async function updateVerifiedAttributeExtensionDateLogic({
     (att) => att.id === attributeId
   );
 
-  assertVerifiedAttributeExistsInTenant(tenantId, attribute, tenant);
+  assertVerifiedAttributeExistsInTenant(attributeId, attribute, tenant);
```

Comment:
@rGregnanin With these changes, I spotted this error! Can you double-check?

Reply:
You're right, thanks for the fix! 😄